src/blob_storage.cc

```diff
@@ -80,33 +136,22 @@ void BlobStorage::GetObsoleteFiles(std::vector<std::string>* obsolete_files,
     // visible to all existing snapshots.
     if (oldest_sequence > obsolete_sequence) {
       // remove obsolete files
-      auto p = files_.find(file_number);
-      assert(p != files_.end());
-      file_dropped++;
-      file_dropped_size += p->second->file_size();
-      files_.erase(p);
-
-      file_cache_->Evict(file_number);
-
+      auto b = RemoveFile(file_number);
```

Comment:
```
bool __attribute__((__unused__)) removed = RemoveFiles(file_number);
assert(removed);
```
`__attribute__((__unused__))` is to avoid unused variable warning in release build.